network_diffusion/tpn/functions.py

```diff
@@ -0,0 +1,50 @@
+"""Script with functions for driver node selections."""
+
+from typing import Any, List, Set
+
+from network_diffusion.mln.actor import MLNetworkActor
+from network_diffusion.mln.mlnetwork import MultilayerNetwork
+
+
+def compute_driver_nodes(net: MultilayerNetwork) -> List[MLNetworkActor]:
```

Comment:
I think we should provide at least an unittest of this function. I can implement it, but please provide me with a minimal working example how you're using it @uniooo